be/src/column/map_column.cpp

```diff
@@ -714,15 +714,23 @@ void MapColumn::remove_duplicated_keys(bool need_recursive) {
 
     uint32_t new_offset = 0;
     for (auto i = 0; i < size; ++i) {
+        std::unordered_map<uint32_t, std::vector<uint32_t>> key_hash_to_offsets;
```

Comment:
https://en.cppreference.com/w/cpp/container/unordered_multimap
You can use std unordered_multimap directly.

Reply:
set is enough?

Reply:
Done.